R/decostand.R

```diff
@@ -260,3 +277,348 @@
         x[abs(x) < sqrt(.Machine$double.eps)] <- 0
     x
 }
+
+
+
+
+
+# .OptSpace : an algorithm for matrix reconstruction from a partially revealed set
```

Comment:
This is not necessary, but I'd prefer doubled ## for comments on their own line.

Reply:
done